e3sm_diags/parser/zonal_mean_2d_stratosphere_parser.py

```diff
@@ -1,14 +1,11 @@
 from e3sm_diags.parameter.zonal_mean_2d_stratosphere_parameter import (
     ZonalMean2dStratosphereParameter,
 )
-from e3sm_diags.parser.zonal_mean_2d_parser import ZonalMean2dParser
+from e3sm_diags.parser.core_parser import CoreParser
 
 
-class ZonalMean2dStratosphereParser(ZonalMean2dParser):
+class ZonalMean2dStratosphereParser(CoreParser):
     def __init__(self, *args, **kwargs):
-        if "parameter_cls" in kwargs:
-            super().__init__(*args, **kwargs)
-        else:
-            super().__init__(
-                parameter_cls=ZonalMean2dStratosphereParameter, *args, **kwargs
-            )
+        super().__init__(  # type: ignore
+            parameter_cls=ZonalMean2dStratosphereParameter, *args, **kwargs
+        )
```

Comment:
I removed subclassing of `ZonalMean2dParser` because it was causing issues in the test suite where `parameter_cls` was being set twice (one here and one in the parent class).

Reply:
I'll need to add an `add_arguments()` method with the same args as `ZonalMean2dParser`